source/fundamentals/logging.txt

```diff
@@ -0,0 +1,477 @@
+               </Console>
+            </Appenders>
+            <Loggers>
+               <Root level="info">
+                     <AppenderRef ref="Console"/>
+               </Root>
+            </Loggers>
+         </Configuration>
+
+      To test that your logger configuration was successful, run the following
+      code.
+
+      .. include:: /includes/fundamentals/logging.rst
+
+      You should see output that resembles the following.
+
+      .. code-block:: none
+
+         ...
+         10:14:57.633 [cluster-ClusterId{value=<your cluster id>, description='null'}-<your connection uri>] INFO  org.mongodb.driver.cluster - Discovered replica set primary <your connection uri>
+         10:14:57.790 [main] INFO  org.mongodb.driver.connection - Opened connection [connectionId{localValue:7, serverValue:<your server value>}] to <your connection uri>
+      
+      For more information on configuring Log4j2, see the official 
+      `Log4j2 configuration guide
+      <https://logging.apache.org/log4j/2.x/manual/configuration.html>`__.
+
+Logger Names
+------------
+
+Your logger uses logger names to help organize different logging events. Logger
+names are strings that form a hierarchy. A logger is an ancestor of another logger if
+its name followed by a ``"."`` is a prefix of the other logger's name. For example,
+``"grandparent"`` is an ancestor of ``"grandparent.parent"`` which is an
+ancestor of ``"grandparent.parent.child"``.
+
+For a concrete example, this is what a logger hierarchy looks like in code.
+
+.. code-block:: java
+
+   import org.slf4j.Logger;
+   import org.slf4j.LoggerFactory;
+
+   ...
+
+   Logger logger_parent = LoggerFactory.getLogger("parent");
+   Logger logger_child = LoggerFactory.getLogger("parent.child");
+
+A logger inherits the properties of its ancestor logger and can define
+its own. You can think of this as similar to class inheritance in Java. 
+
+The MongoDB Java driver defines the following logger names to organize different
+logging events in the driver. Here are the logger names defined in the driver
+and the logging events they correspond with. 
+
+* ``org.mongodb.driver.authenticator`` : authentication
+* ``org.mongodb.driver.client`` : events related to ``MongoClient`` instances
+* ``org.mongodb.driver.cluster`` : monitoring of MongoDB servers
+* ``org.mongodb.driver.connection`` : connections and connection pools
+* ``org.mongodb.driver.connection.tls`` : TLS/SSL
+* ``org.mongodb.driver.operation`` : operations, including logging related to automatic retries
+* ``org.mongodb.driver.protocol`` : commands sent to and replies received from MongoDB servers
+* ``org.mongodb.driver.uri`` : connection string parsing
+* ``org.mongodb.driver.management`` : JMX (Java Management Extensions)
+
+Example - Names
+~~~~~~~~~~~~~~~
+
+This example shows how to change the log level for a specific driver logger.
+We set the root logger to OFF and the ``org.mongodb.connection`` logger to
```

Comment:
org.mongodb.connection -> org.mongodb.driver.connection

Reply:
Thanks for the note. Fix: https://github.com/mongodb/docs-java/pull/76/commits/23b4d91497076f64afd53a1813cf90c43f741d28